app/components/transformations/booking.rb

```diff
@@ -0,0 +1,32 @@
+module Transformations
```

Comment:
We should reflect somewhere in the namespace that this object is concerned with transforming Booking Bug booking data.